react/Badge/Readme.md

```diff
@@ -33,7 +33,7 @@ const initialVariants = [
         horizontal: variant.left ? "left" : "right",
       }}
       withBorder={variant.withBorder}
-      overlap={variant.overlap ? "circle" : "rectangle"}
```

Comment:
Since which versions those new properties (circular / rectangular) are supported? Are we sure their are available on 4.12.0 since our peerDep is 4.12.0? https://github.com/cozy/cozy-ui/blob/master/package.json#L177

Reply:
Historique du fichier : https://github.com/mui/material-ui/commits/v4.x/packages/material-ui/src/Badge/Badge.js

Les versions historiques `circle / rectangle` ont été introduites le 31 aout 2019 : https://github.com/mui/material-ui/commit/e97d4740dadda0454b5e1e8ba367df1261132890

Les nouvelles `circular / rectangular` le 7 aout 2020 par commit https://github.com/mui/material-ui/commit/f00c44131b18d7700b3834ff0c0c75d7be19b79b et le 06 juillet 2021 dans la version 4.12.0 : https://github.com/mui/material-ui/releases/tag/v4.12.0

Le warn est introduit dans la 4.12.4 : https://github.com/mui/material-ui/releases/tag/v4.12.4